gui/syncthing/core/eventService.js

```diff
@@ -18,14 +18,43 @@ angular.module('syncthing.core')
                 return;
             }
             $rootScope.$broadcast(self.ONLINE);
-
+            //GREEN
+            $("link[rel='shortcut icon']").attr("href","assets/img/favicon_idle.png")
```

Comment:
Also, this specific modification will always be overriden by the else BLUE branch.